test/entity/entity_test.go

```diff
@@ -201,7 +226,7 @@ func ValidateLogEntity(t *testing.T, logGroup, logStream string, end *time.Time,
 		t.Fatalf("application log group used for entity validation doesn't exist: %s", logGroup)
 	}
 
-	begin := end.Add(-sleepForFlush * 2)
+	begin := end.Add(-2 * time.Minute)
```

Comment:
We increased the wait time to be 4 minutes, and we query 2 minutes before the end time. The following is the justification:
- 1 minute for agent to get and store AppSignal attributes in server extension
- 1 minute for FluentBit to refresh and get the service name from agent
- 1 minute for GetQueryResults to be available
- Last minute as buffer period since we need an extra minute to query the period when GetQueryResult is available